CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.1.0 (March 4, 2021)
+
+* Allow tokens to be revoked and manually refreshed.
+
+  PR #39 - https://github.com/procore/ruby-sdk/pull/39
+
+  *Nate Baer*
```

Comment:
Can you pull this out? The release commit takes care of version bumping. For now change the header to `## Unreleased`